vcd/resource_vcd_org_vdc.go

```diff
@@ -389,6 +429,48 @@ func resourceVcdVdcDelete(d *schema.ResourceData, meta interface{}) error {
 	return nil
 }
 
+func manageMetaData(d *schema.ResourceData, meta interface{}) error {
```

Comment:
A very generic name it seems. The name doesn't hint what it actually does. You only know that it's "doing something with metadata"

Reply:
changed